pkg/apis/operator/v1alpha1/catapult_types.go

```diff
@@ -40,19 +40,6 @@ type CatapultSpec struct {
 	Paused PausedFlagType `json:"paused,omitempty"`
 }
 
-// PausedFlagType represents a enable/disable flag
```

Comment:
why is this removed? This PR, at least by the title, has nothing to do with the catapult.

Reply:
it's a common type for all operators, it was in `catapult` only because was implemented first for `catapult` but in this PR it was moved to `types.go`